ddsrecorder/test/blackbox/mcap/McapFileCreationTestWithController.cpp

```diff
@@ -0,0 +1,585 @@
+// Copyright 2023 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

Comment:
Please rename this file to something more descriptive (controller is not present). StateTransitionTest?